pkg/webhook/webhook.go

```diff
@@ -524,6 +532,18 @@ func (wh *webhook) validateNamespaceDeleteRequest(req *v1beta1.AdmissionRequest)
 		}
 		return response
 	}
+
+	err = wh.kubeClient.AdmissionregistrationV1().
+		ValidatingWebhookConfigurations().
+		Delete(validatorWebhook, &metav1.DeleteOptions{})
+	if err != nil {
+		response.Allowed = false
+		response.Result = &metav1.Status{
+			Message: err.Error(),
+		}
+		return response
+	}
+
```

Comment:
this change only delete the validatingwebhookconfiguration if request came to delete the openebs namespace.
But we have to think about how to handle the deletion in case of  only cstor specific components has been removed from the cluster in openebs namespace , but other components exists like localPV, in such cases user has to delete the webhook config manually.